### DISCUSSION TITLE
Directory specifications: "Sort Name" and different directory in $HOME

### DISCUSSION THREAD

**Discussion by keeganjk** [beetbox/beets#3910] [Q&A] [answered]
I have two things which I have had difficulty figuring out with `beets`. 

One of these things is using the MusicBrainz "Sort Name" as opposed to the artist name in the directory that music is placed. For example, I have:
```yaml
paths: 
    default: artist/$albumartist/$original_year~$album%aunique{}/$track~$title
```
in my `config.yaml`. Is it possible for me to use the "Sort Name" (e.g. "Beatles, The" instead of "The Beatles") in the name of the directory, so that I do not have a bunch of things starting with "The" or other articles all in one place?

Another issue I have is importing the music to alternate directories in the home directory. It seems like it would be very simple and possible for me to do this, but the music is always put into `~/Music` instead of `~/aud/mus`, which I would prefer—for now, I simply symlink `~/Music` to `~/aud/mus`, but I'd prefer that things be put there in the first place.

**Comment by sampsyo** [comment, marked as answer]
Hello! You might try `$artist_sort` or `$albumartist_sort` to begin with (they're populated from MusicBrainz, but you might need to set them otherwise for other data sources).

For the other thing, take a look at the [directory](https://beets.readthedocs.io/en/stable/reference/config.html#directory) config option.

**Reply by keeganjk** [reply]
I don't know how powerful YAML is or if it can be "scripted"; is there a way to do `$albumartist_sort` and fall back to `$albumartist` if nonexistent?

**Reply by keeganjk** [reply]
Okay, it appears like everything has a sort name, which may be the artist's normal name, or a mutation that involved romanization or moving articles to the end. That should probably be good.
